setup.py

```diff
@@ -48,4 +51,32 @@ def setup_optimism_repo():
 
     print("Successfully built the optimism repository.")
 
+
 ####################################################################################################
+
+def setup_op_geth_repo():
+    """
+    Clone the op-geth repository and build it.
+    """
+    github_url = "https://github.com/ethereum-optimism/op-geth.git"
+    git_tag = "v1.101106.0"
```

Comment:
We should an issue to add functionality to update the optimism and op-geth repo to new version.

Reply:
https://github.com/0xFableOrg/simple-op-stack-rollup/issues/29